src/libraries/Microsoft.Win32.SystemEvents/src/Microsoft/Win32/SystemEvents.cs

```diff
@@ -464,39 +459,43 @@ private void Dispose()
         /// </summary>
         private static void EnsureSystemEvents(bool requireHandle)
         {
-            if (s_systemEvents == null)
+            if (s_systemEvents is not null)
             {
-                lock (s_procLockObject)
+                return;
+            }
+
+            lock (s_procLockObject)
+            {
+                if (s_systemEvents is not null)
                 {
-                    if (s_systemEvents == null)
-                    {
-                        // Create a new pumping thread.  We always create one even if the current thread
-                        // is STA, as there are no guarantees this thread will pump nor still be alive
-                        // for the desired duration.
+                    return;
+                }
```

Comment:
Are these just stylistic changes?

Reply:
Yes, this is just a style change. I inverted a couple if statement in this method and changed `==` to use `is`